hibernate-core/src/test/java/org/hibernate/jpa/test/criteria/basic/PredicateTest.java

```diff
@@ -204,115 +125,4 @@ public void testComplicatedNotAnd() {
 		em.getTransaction().commit();
 		em.close();
 	}
-
```

Comment:
The reason is the testing case fails for the current Predicate implementation. Needs to discuss with Steve on this to see whether we need to tweak the testing case.
In v5 when we negate a compound predicate, we reverse its operand (from OR to AND or from AND to OR). Now in v6 we stop doing this and simply end up with a new wrapper. Not sure whether we wanna do this for immutability reason. Needs to confirm with Steve for he implemented this previously.

Reply:
@sebersole could you shed some lights on this?
```
orderCriteria.select( orderRoot );
Predicate p1 = builder.equal( orderRoot.get( "id" ), "order-1" );
Predicate p2 = builder.equal( orderRoot.get( "id" ), "order-2" );
Predicate compoundPredicate = builder.and( p1, p2 ).not();
// a negated AND should become an OR
assertEquals( Predicate.BooleanOperator.OR, compoundPredicate.getOperator() );
```
The above assertion would fail because we seem to have changed the implementation without reversing as in v5.